bridge_adaptivity/module/mixins/models.py

```diff
@@ -0,0 +1,33 @@
+# coding: utf-8
+from uuid import uuid4
+
+
+class WithUniqueSlugMixin(object):
```

Comment:
It seems that this mixing can be omitted with the usage of `UUIDField` https://docs.djangoproject.com/en/2.0/ref/models/fields/#uuidfield